src/partials/header.html

```diff
@@ -1,47 +1,61 @@
-            <svg width="44" height="44">
-              <use href=""></use>
-            </svg>
-          </a>
-        </li>
-        <li>
-          <a class="social-media-link" href="#" target="_blank">
-            <svg width="44" height="44">
-              <use href=""></use>
-            </svg>
-          </a>
-        </li>
-        <li>
-          <a class="social-media-link" href="#" target="_blank">
-            <svg width="44" height="44">
-              <use href=""></use>
-            </svg>
-          </a>
-        </li>
-      </ul>
+    <nav class="nav-menu">
+      <a class="logo" href="./index.html">english<span class="logo-part">excellence</span>.</a>
+      <div class="menu-container js-menu-container" id="mobile-menu">
+        <button class="close-button js-close-menu">
+          <svg class="icon-close">
+            <use href="./images/icons.svg#icon-cross"></use>
+          </svg>
+        </button>
+        <ul class="header-nav-list mobile-menu">
+          <li class="header-nav-link mobile-menu-item">
+            <a class="js-close-menu" href="#about">About us</a>
+          </li>
+          <li class="header-nav-link mobile-menu-item">
+            <a class="js-close-menu" href="#lessons">Our lessons</a>
+          </li>
+          <li class="header-nav-link mobile-menu-item">
+            <a class="js-close-menu" href="#teachers">Teachers</a>
+          </li>
+          <li class="header-nav-link mobile-menu-item">
+            <a class="js-close-menu" href="#reviews">Reviews</a>
+          </li>
+          <li class="header-nav-link mobile-menu-item">
+            <a class="js-close-menu" href="#application">Contact us</a>
+          </li>
+        </ul>
+
+        <ul class="social-media-list">
+          <li class="social-media-item">
+            <a href="https://www.facebook.com/goITclub/" target="_blank">
+              <svg class="social-media-icon">
+                <use href="./images/icons.svg#icon-facebook"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="social-media-item">
+            <a href="https://www.instagram.com/goitclub/" target="_blank">
+              <svg class="social-media-icon">
+                <use href="./images/icons.svg#icon-instagram"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="social-media-item">
+            <a href="https://www.youtube.com/c/GoIT" target="_blank">
+              <svg class="social-media-icon">
+                <use href="./images/icons.svg#icon-youtube"></use>
+              </svg>
+            </a>
+          </li>
+        </ul>
+
+      </div>
       <button class="menu-toggle js-open-menu" aria-expanded="false" aria-controls="mobile-menu">
-        <svg width="32" height="32">
-          <use href=""></use>
+        <svg class="burger-icon">
+          <use href="./images/icons.svg#icon-burger"></use>
         </svg>
       </button>
     </nav>
-  </div>
```

Comment:
Looks like you lost the closing tag for <div class="container"> 
Also, I think we can just use <header class="container"> to avoid unnecessary nesting.

Reply:
Thank you! Applied proposed changes.